Input.cs

```diff
@@ -1,35 +1,114 @@
 ﻿using System;
+using System.IO;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using CMP1903M_Assessment_1.Debugging;
 
 namespace CMP1903M_Assessment_1
 {
-    public class Input
+    /// <summary>
+    /// Handles taking and verifying user input
+    /// </summary>
+    public static class Input // felt like this should be static rather than an instance
     {
-        //Handles the text input for Assessment 1
-        string text = "nothing";
-        
-        //Method: manualTextInput
-        //Arguments: none
-        //Returns: string
-        //Gets text input from the keyboard
-        public string ManualTextInput()
+        /// <summary>
+        /// Gets text input from the keyboard
+        /// </summary>
+        /// <returns>Returns the the text from keyboard input</returns>
+        public static string ManualTextInput()
         {
+            string text = "";
 
-            return text;
+            // TODO: Use * to end the input
+            Console.WriteLine("Manual Text Input (Supports multiline using \'\\n\')");

```

Comment:
This line could be simplified:
- No need to escape single quotes because string literal encapsulated by double quotes.
- Can place "@" before string literal to enforce literal mode (no escape characters necessary) 

```suggestion
            Console.WriteLine(@"Manual Text Input (Supports multiline using '\n')");
```